src/mare/compiler.cr

```diff
@@ -1,4 +1,19 @@
 module Mare::Compiler
+  class CompilerOptions
+    property release
+    property no_debug
+    property binary_name
+
+    DEFAULT_BINARY_NAME = "main.out"
```

Comment:
Why `main.out` instead of `main`?

Reply:
I thought that main is less common name coz we will overwrite this file.